cpp/config/Make.rules

```diff
@@ -98,7 +98,7 @@ endif
 # directory.
 #
 cpp11_cppflags          = -DICE_CPP11_MAPPING
-ifneq ("OK", $(shell $(CC) -E config/cplusplus_check.cpp 2> /dev/null && echo "OK"))
+ifeq ($(shell $(CC) -E config/cplusplus_check.cpp 1> /dev/null 2>&1 && echo "OK"),)
```

Comment:
Why not just use the exit status?

```suggestion
ifneq ($(shell $(CC) -E config/cplusplus_check.cpp 1> /dev/null 2>&1; echo $$?),0)
``

Reply:
No particular reason, I was more familiar with the other style